Doist/typist
# feat: Add `createEmptyParagraphEnd` command

## Overview

Adds a new command, `createEmptyParagraphEnd`, which creates an empty paragraph always at the end of the document.

This command that can be invoked like this:

- `editor.chain().focus().createEmptyParagraphEnd().run()`

This is just a helpful command that doesn't come built-in with Tiptap.

## PR Checklist


## Test plan

- Open the preview Storybook deployed to Netlify
- Open the `Rich-text → Functions → Commands` story
- Type a multiline document (or copy and paste something)
- Place the text cursor in multiple locations, and in between press the `createEmptyParagraphEnd` button
    - [ ] Observe that regardless of the text cursor position, an empty paragraph is always created at the end of the document
- Open the `Plain-text → Functions → Commands` story
- Type a multiline document (or copy and paste something)
- Place the text cursor in multiple locations, and in between press the `createEmptyParagraphEnd` button
    - [ ] Observe that regardless of the text cursor position, an empty paragraph is always created at the end of the document
